src/resources/views/configuration/users/list.blade.php

```diff
@@ -8,91 +8,155 @@
                   </span>
-              @else
-                <span class="label label-danger">
+                  @else
+                    <span class="label label-danger">
                     {{ trans('web::seat.disabled') }}
                   </span>
-              @endif
-            </td>
-            <td data-order="{{ $user->refresh_token ? 1 : 0 }}">
+                  @endif
+                </td>
+                <td data-order="{{ $user->refresh_token ? 1 : 0 }}">
+                  @if($user->refresh_token)
+                    <i class="fa fa-check text-success"></i>
+                  @else
+                    <i class="fa fa-exclamation-triangle text-danger"></i>
+                  @endif
+                </td>
+                <td>{{ $user->email }}</td>
+                <td data-order="{{ $user->last_login }}">
+              <span data-toggle="tooltip" title="" data-original-title="{{ $user->last_login }}">
+                {{ human_diff($user->last_login) }}
+              </span>
+                </td>
+                <td>{{ $user->last_login_source }}</td>
+                <td>{{ count($user->group->roles) }}</td>
+                <td>
+                  <div class="btn-group">
+                    <a href="{{ route('configuration.users.edit', ['user_id' => $user->id]) }}" type="button"
+                       class="btn btn-warning btn-xs">
+                      {{ trans('web::seat.edit') }}
+                    </a>
+                  </div>
+
+                  @if(auth()->user()->id != $user->id)
+                    <div class="btn-group">
+                      <a href="{{ route('configuration.users.delete', ['user_id' => $user->id]) }}" type="button"
+                         class="btn btn-danger btn-xs confirmlink">
+                        {{ trans('web::seat.delete') }}
+                      </a>
+                      <a href="{{ route('configuration.users.impersonate', ['user_id' => $user->id]) }}" type="button"
+                         class="btn btn-success btn-xs">
+                        {{ trans('web::seat.impersonate') }}
+                      </a>
+                    </div>
+                  @else
+                    <em class="text-danger">(This is you!)</em>
+                  @endif
+                </td>
+          </tr>
+        @endforeach
+        @continue
+        @endif
+
+        <td>{{ $user_group->main_character->name }}</td>
+        <td>
+          @if($user_group->isActive())
+            <span class="label label-success">
+                    {{ trans('web::seat.enabled') }}
+                  </span>
+          @else
+            <span class="label label-danger">
+                    {{ trans('web::seat.disabled') }}
+                  </span>
+          @endif
+        </td>
+        <td data-order="{{$user_group->mainCharacterUser->refresh_token ? 1 : 0 }}">
+          @foreach($user_group->users as $user)
+            <li>
```

Comment:
`li` with no parent `ul` ? bubu
about list style : `<ul class="list-style-none">`

Reply:
absolutely did implement it just now:

![picture](https://i.imgur.com/pj5OI55.png)